src/main/java/teammates/common/util/Const.java

```diff
@@ -259,7 +263,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String STUDENT_COURSE_PROFILE = "Your profile in this course";
         public static final String STUDENT_COURSE_DETAILS = "View and edit information regarding your team";
     
-        public static final String STUDENT_FEEDBACK_SESSION_STATUS_AWAITING = "The session is not open for submission at this time. It is expected to open later.";
+        public static final String STUDENT_FEEDBACK_SESSION_STATUS_AWAITING =
+                "The session is not open for submission at this time. It is expected to open later.";
```

Comment:
maybe we should have a rule that if the whole thing cannot fit in a line, the right hand side should always be on a new line

so instead of
~~~
public static final String A_MEDIUM_LENGTH_VAR = "..............................."
                                               + "..............................."
~~~

or

~~~
public static final String A_MEDIUM_LENGTH_VAR = "..............................."
        + "......................................................................"
~~~

it must be

~~~
public static final String A_MEDIUM_LENGTH_VAR = 
        + "......................................................................"
        + "..............................."
~~~